ui.frontend/src/main/webpack/components/_breadcrumb.scss

```diff
@@ -1,5 +1,20 @@
-.cmp-breadcrumb {}
-.cmp-breadcrumb__list {}
-.cmp-breadcrumb__item {}
-.cmp-breadcrumb__item--active {}
-.cmp-breadcrumb__item-link {}
\ No newline at end of file
+.cmp-breadcrumb__list {
+    display: flex;
+    column-gap: 5px;
+    list-style: none;
+}
+
+.cmp-breadcrumb__item {
+    text-transform: uppercase;
+    font-weight: $font-semibold;
```

Comment:
font weight should not be bold for all items. Please review colleagues work and fix